perf/replicache.ts

```diff
@@ -359,16 +359,15 @@ export function benchmarkWriteSubRead(opts: {
     async run(bencher: Bencher) {
       const keys = Array.from({length: numKeys}, (_, index) => makeKey(index));
       const sortedKeys = keys.sort();
-      const data: Map<string, TestDataObject> = new Map(
-        keys.map(key => [key, jsonObjectTestData(valueSize)]),
-      );
+      const data: Map<string, TestDataObject> = new Map();
```

Comment:
My theory here was that holding onto these random objects for less time would reduce GC pressure, but I'm not 100% sure it did anything.

Reply:
They aren't held for much less time right?  in init below, we put them into replicache, which will keep a reference to them for the full test.

Reply:
Ah good point, forgot about that. I'll abandon this one.